libraries/AP_HAL_ChibiOS/GPIO.cpp

```diff
@@ -147,25 +183,16 @@ bool GPIO::_attach_interrupt(ioline_t line, AP_HAL::Proc p, uint8_t mode)
         osalSysUnlock();
     }
 
+    if (!p) {
+        chmode = PAL_EVENT_MODE_DISABLED;
+    }
+
     palDisableLineEvent(line);
     palEnableLineEvent(line, chmode);
-    palSetLineCallback(line, p?pal_interrupt_cb:nullptr, (void*)p);
```

Comment:
we need to check if there is a conflict with another pad, by calling pal_lld_get_line_event() and checking if the cb field is set